.github/workflows/ci.yml

```diff
@@ -28,13 +28,19 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install -r dev-requirements.txt
+          pip install pytest-cov
           pip install -e .
 
       - name: Run mypy
         run: mypy .
 
       - name: Run tests
-        run: pytest
+        run: pytest --cov --cov-branch --cov-report=xml
```

Comment:
`--cov` is already included in the pyproject.toml (although the target needs updating).

What does `--cov-branch` do?

Reply:
Enables branch coverage, which I presume is a record on the coverage per branch.